frontend/src/components/ProjectHistoryTimeline.css

```
@@ -0,0 +1,241 @@
+}
+
+.project-history-counts,
+.project-history-time-basis,
+.project-history-warning,
+.project-history-boundary {
+  margin: 0;
+}
+
+.project-history-warning,
+.project-history-boundary {
+  border-inline-start: 0.25rem solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)